app/src/evaluation/literalai_exporter.py

```diff
@@ -133,7 +132,9 @@ def from_lai_thread(
 def convert_to_qa_rows(project_id: str, threads: list[Thread]) -> list[QARow]:
     qa_pairs = []
     for th in threads:
-        assert th.steps
+        if not th.steps:
+            logger.warning("Thread %r has no steps", th.id)
+            continue
```

Comment:
Due to LiteralAI problem: https://nava.slack.com/archives/C06DP498D1D/p1741620115782139